controllers/rack.go

```diff
@@ -472,6 +502,9 @@ func (r *SingleClusterReconciler) scaleDownRack(
 		"ScaleDown AerospikeCluster statefulset", "desiredSz", desiredSize,
 		"currentSz", *found.Spec.Replicas,
 	)
+	r.Recorder.Event(r.aeroCluster, corev1.EventTypeNormal, "RackScaleDown",
```

Comment:
It's better to place this event and the above log when scale down is needed. Basically after this condition
`if *found.Spec.Replicas <= desiredSize {
		return found, reconcileSuccess()
	}`